Resources/views/CRUD/edit_orm_one_to_many.html.twig

```diff
@@ -106,7 +106,7 @@ file that was distributed with this source code.
                                 </div>
 
                                 {% if nested_group_field['_delete'] is defined %}
-                                    {{ form_row(nested_group_field['_delete']) }}
+                                    {{ form_row(nested_group_field['_delete'],  { 'label': ('action_delete'|trans({}, 'SonataAdminBundle')) } ) }}
```

Comment:
IMO we don't need a break here, because we have many indentions caused by the html structure. 

Btw. we have many lines in this file who have much longer lines.